app/assets/stylesheets/pages/_show.scss

```diff
@@ -0,0 +1,38 @@
+.card-product {
```

Comment:
Could you move everything in this file to the _card.scss file in the components folder instead? Since all this code concerns card components and not really the structure of the page itself.
Note: Sandra already has some of the CSS from the UI kit in that file, please make sure there's no duplication